src/gf-oemid

```diff
@@ -18,6 +18,29 @@ oemid=$3
 
 tmpd=$(mktemp -td gf-oemid.XXXXXX)
 tmp_dest=${tmpd}/box.img
+
+# Work around for https://github.com/coreos/coreos-assembler/issues/198
```

Comment:
can we get a more comprehensive comment here? Maybe something like "performance monitoring for nested virtualization doesn't work reliably on different virt platforms (VMWare, VirtualBox, etc) so just disable the pmu".

Also can we change the title of #198 to more accurately reflect the issue (i.e. pmu)?

Reply:
Done